feathr-example/README.md

```diff
@@ -1 +1,73 @@
+        1. Go to `Monitor` in the left pane and `Apache Spark applications` pane. In this pane you could verify the status of the job that is running to update the Spark pool.
+
+## Notes on the notebooks
+There are several gotchas on these notebooks:
+1. These notebooks only utilized the `pre-processing` feature of `feathr`.
+1. If anchors have aggregated and a created on-the-fly column features each definition should have separate datasource - even. though they would pointing to the same csv.
+1. If anchors have pass-thru and a created on-the-fly column features, they could be on the same datasource.
+1. Can't mix a pass-thru and a window aggregated/transformed feature definition.
+1. Pre processing methods
+    1. Expects a Spark Dataframe as input and output.
+        1. The `feast` implementation uses Panda Dataframe, so, to minimize code conversion between the the two, inside the pre processing method the passed dataframe (Spark) would be converted into Panda dataframe prior to calling the Panda methods (same method from the `feast` implementation). After the Panda dataframe manipulation, this dataframe would be converted back to Spark dataframe as a return value.
+    1. Can't do method chaining (calling another method) inside the pre processing method. To overcome this limitation, a nested function is used (function within a function). So the `feathr` spark job could execute the pre processing method, then this method would in turn call other methods that is inside the preprocessing methods.
+        ``` python
+        def bureau_preprocessing(df: DataFrame) -> DataFrame:
+            
+            def bureauBalanceRollingCreditLoan(df):
+                PASS
+            
+            df = bureauBalanceRollingCreditLoan(df)
+            return df
+        ```
+    1. Importing Python packages has to be inside the pre processing methods.
+        ``` python
+        def bureau_preprocessing(df: DataFrame) -> DataFrame:
+            import datetime
+            import pandas as pd
+            from pyspark import sql
+            from sklearn.preprocessing import OneHotEncoder, MinMaxScaler
+
+            def bureauBalanceRollingCreditLoan(df):
+                PASS
+            
+            df = bureauBalanceRollingCreditLoan(df)
+            return df
+        ```
+    1. During feature definition, `feast` assumes that the `feature`(column) is in the datasource. So it's is important to check if the output features of the pre processing methods contains the fields defined in the feature defition. This is the reason that in the pre processing method, there is a Panda merge method call. And subsequently, after this method call, a column rename is called. This is because that during the merge call, if there is a same column name, Panda would rename the duplicates with a suffix of `_x` and the other with `_y`. Removing the `_x` suffix (only) would retain the original column name that is being referenced in the feature definition.
```

Comment:
Is it supposed to say `feast` or `feathr`?

Reply:
Changed this to `feathr`. Thanks.